### PR TITLE
[New Term] C++ Math Functions: `fmin()`

### LINKED ISSUE #965: [Term Entry] C++ Math Functions: `fmin()`
### Reason/inspiration (optional)

Under `cpp/concepts/math-functions/terms/`, create a new term entry for the `fmin()` function.

- In the `terms/` folder, make a new folder called `fmin`
- Inside the `fmin` folder, create a new markdown file called `fmin.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Let's include an "Example" section with a \```cpp code snippet.
- Lastly, let's include a "Codebyte Example" with a runnable \```codebyte/cpp

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by borntofrappe**
### Description

Closes #965 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/cpp/concepts/math-functions/terms/fmin/fmin.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+Title: 'fmin()'
+Description: 'Returns the smaller of the two input arguments.'
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/fmin/fmin.md:3]:
```suggestion
Description: 'Returns the smaller of two arguments.'
```

File: content/cpp/concepts/math-functions/terms/fmin/fmin.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`fmin()`** function returns the smaller of the two input arguments.
```
Comment (SSwiniarski) [inline content/cpp/concepts/math-functions/terms/fmin/fmin.md:15]:
```suggestion
The **`fmin()`** function returns the smaller of two arguments.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@borntofrappe, I did a first review and left a couple of comments.

**Review by SSwiniarski — APPROVED** [review]
@borntofrappe, looks like we're ready for a second review.

**Comment by yangc95** [comment]
@borntofrappe I'll be taking this PR on for a second review~

File: content/cpp/concepts/math-functions/terms/fmin/fmin.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+The **`fmin()`** function returns the smaller of two arguments.
+
+## Syntax
+
+```pseudo
+fmin(x, y)
+```
+
+The parameters `x` and `y` are usually a numeric [data type](https://www.codecademy.com/resources/docs/cpp/data-types) ranging from `-INFINITY` (negative) to `INFINITY` (positive). If one of the parameters is `NAN`, then the other parameter is returned.
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/fmin/fmin.md:23]: - Similar to the open PR for `fmax()` function, can we change  to to "negative `INFINITY` to positive `INFINITY`."
- Change `NAN` to `NaN`

File: content/cpp/concepts/math-functions/terms/fmin/fmin.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+This produces the following output:
+
+```shell
+The smaller value between 3 and 5 is 3
+```
+
+## Codebyte Example
+
+The following example is runnable and returns the smaller value between `-32.123` and `-32.231`:
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/fmin/fmin.md:50]:
```suggestion
The following example returns the smaller value between `-32.123` and `-32.231`:
```

File: content/cpp/concepts/math-functions/terms/fmin/fmin.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+```codebyte/cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+  double x = -32.123;
+  double y = -32.231;
+  double result;
+  result = fmin(x, y);
+  std::cout << "The smaller value between " << x << " and " << y << " is " << result << "\n";
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/fmin/fmin.md:61]: Can we add a "." period at the end of this output before the new line?

File: content/cpp/concepts/math-functions/terms/fmin/fmin.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+```cpp
+#include <iostream>
+#include <cmath>
+
+int main() {
+  double x = 3;
+  double y = 5;
+  double result;
+  result = fmin(x, y);
+  std::cout << "The smaller value between " << x << " and " << y << " is " << result << "\n";
```
Comment (yangc95) [inline content/cpp/concepts/math-functions/terms/fmin/fmin.md:38]:
```suggestion
  std::cout << "The smaller value between " << x << " and " << y << " is " << result << "." << "\n";
```

**Review by yangc95 — CHANGES_REQUESTED** [review]
Hi @borntofrappe, I've added some new comments/suggestions.

Please tag me once you're done making edits or if you have any questions!

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
Thanks for this entry @borntofrappe 🥳

live link: https://www.codecademy.com/resources/docs/cpp/math-functions/fmin
Screen Shot 2022-10-25 at 8 35 46 PM

cc: @SSwiniarski

**Pull request merged**